examples/tutorials/setup_poc.ipynb

```diff
@@ -34,91 +36,129 @@
    "id": "87989e99",
    "metadata": {},
    "source": [
-    "### Preparing the POC environment\n",
-    "Before running POC mode, there are a couple important environment variables that should be set.\n",
+    "> If you are the NVFLARE user prior to 2.4.0, you might noticed two things\n",
+    "> * poc command is now separated into sub-commands\n",
+    "> * the sub-commands such as \"prepare\", \"start/stop\" etc. are different from previous version where \"--\" is removed for command, such as \n",
+    ">   \"--prepare\" becomes \"prepare\" and \"--start\" becomes \"start\" etc. \n",
+    "\n",
+    "We can further look at the help description for each command\n",
```

Comment:
Do we want to mention these points here?
1. poc command is now separated into sub-commands
2. the sub-commands such as \"prepare\", \"start/stop\" etc. are different from previous version where \"--\" is removed for command